spec/lib/generators/full_generator_test_blobs.rb

```diff
@@ -0,0 +1,193 @@
+# rubocop:disable Metrics/ClassLength, Metrics/MethodLength
+class FullGeneratorTestBlobs
```

Comment:
Perhaps namespace this pseudo-utility/pseudo-fixture under `/spec/lib/generators/utils/` (or something like `/spec/lib/utilities` ) to increase clarity of intent, and also allow a future option to split up into smaller files if this gets unwieldy.

Reply:
Agreed... will make the change :)